packages/headless/src/features/facets/facet-api/request.ts

```diff
@@ -16,7 +19,10 @@ export interface BaseFacetRequest {
    * @default 1000
    */
   injectionDepth: number;
```

Comment:
David noted previously that "scan" isn't the clearest, and I agree. What exactly does this option do? Is it the number of items to look at to find the values that are displayed in the facet, or does it have to do with the result count for each value?

Reply:
I believe this parameter determines how deep the index should search to determine facet value counts. The `1000` default should return accurate counts for most people, but some clients may elect to set a higher number based on their document structure (not sure whether it is number of documents or hierarchy). With a higher number though, the index has to work harder, slowing down the response.

I would ask Daniel Lavoie for more details. He has a strong understanding of the internal workings of the index.

Reply:
Perfect, I've reached out to him and will update this in the next commit following his response 👍🏻